sklearn/preprocessing/tests/test_common.py

```diff
@@ -44,12 +47,17 @@ def test_missing_value_handling(est, func, support_sparse):
     assert np.any(np.isnan(X_test), axis=0).all()
     X_test[:, 0] = np.nan  # make sure this boundary case is tested
 
-    Xt = est.fit(X_train).transform(X_test)
+    with pytest.warns(None) as records:
```

Comment:
Why not using `sklearn.utils.testing.assert_no_warnings`?

Reply:
I wanted to stick to pytest awaiting for this feature: https://github.com/pytest-dev/pytest/issues/1830
The second point is that I find more readable

```python
with pytest.warns(None):
    X_t = est.whatever(X)
```

than

```python
X_t = assert_no_warnings(est, whaterver, X)
```

@ogrisel are you also in favor of `assert_no_warnings`. If yes, 2 vs 1 and I will make the change :)

Reply:
No ok, this is fine as it is.